src/vpn.ts

```
@@ -0,0 +1,28 @@
+interface TimeZone {
+    geoip: string | null | undefined;
+    localtime: string | null | undefined;
+}
+
+export async function getVpnStatus(timeZone: TimeZone): Promise<Object> {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)